src/android/TTS.java

```diff
@@ -412,19 +407,24 @@ private void TTSspeak(Set<Voice> voices, CallbackContext callbackContext, HashMa
         }
 
         if (voice != null) {
-            Timber.d("We've got a voice: " + voice.getName());
+            Timber.d("We've got a voice: %s", voice.getName());
             tts.setVoice(voice);
         } else {
             Timber.d("No voice found.");
         }
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O_MR1) {
+            Timber.d("Setting speech rate to: %s", (float) speechRate * 0.7f);
             tts.setSpeechRate((float) speechRate * 0.7f);
         } else {
+            Timber.d("Setting speech rate to: %s", (float) speechRate);
             tts.setSpeechRate((float) speechRate);
         }
+        Timber.d("Setting pitch to: %s", (float) pitch);
         tts.setPitch((float)pitch);
 
+        Timber.d("TTS isSpeaking: %s", tts.isSpeaking());
+        Timber.d("starting speech: %s", textToRead);
```

Comment:
can we change this to:

```
Timber.d("starting speech: '%s'", textToRead);
```

with `'` marks around `%s`? (that way if its null or empty for whatever reason, its more explicit in the logs)